developer-hub/src/polkadot_sdk/cumulus.rs

```diff
@@ -0,0 +1,132 @@
+//! # Cumulus
+//!
+//! Substrate provides a framework through which a blockchain node and runtime can easily be
+//! created. Cumulus aims to extend the same approach to creation of Polkadot parachains.
+//!
+//! > Cumulus clouds are shaped sort of like dots; together they form a system that is intricate,
+//! > beautiful and functional.
+//!
+//! ## Example: Runtime
+//!
+//! A cumulus based runtime is fairly similar to a normal [FRAME]-based runtime. Most notably, the
+//! following changes are applied to a normal FRAME-based runtime to make it a cumulus-based
```

Comment:
We should probably use the same style in both sentences. Maybe removing the square brackets and capitalising? 
Also, the first sentence might make it look like Cumulus is not FRAME based.
```suggestion
//! A Cumulus-based runtime is fairly similar to other FRAME-based runtimes. Most notably, the
//! following changes are applied to a normal FRAME-based runtime to make it a Cumulus-based
```